tests/unit/cloudwatch.unit.test.ts

```diff
@@ -10,20 +10,20 @@ describe("The CloudWatch class", () => {
 
   it("should send visit metrics", async () => {
     const [visitsToday, oldVisits, openVisits] = [42, 0, 5];
-    expect.assertions(2)
-    await expect(cw.sendVisits(visitsToday, oldVisits, openVisits)).resolves.toBe(`visits: ${visitsToday}, oldVisits: ${oldVisits}, openVisits: ${openVisits}`);
+    expect.assertions(2);
+    await expect(cw.sendVisits(visitsToday, oldVisits, openVisits)).resolves.toBe(undefined);
```

Comment:
I think it is probably best if we return something, here it is undefined and could be for any reasons (function not doing its job for example), etc... I think. Happy to discuss about it.

Reply:
See above explanations on these.

Reply:
This is specifically to expect a void promise.